cypress-io/cypress
# fix: Add missing error message when `req.continue` is used incorrectly

No existing GH issue, found during course of work

### Additional details
Referenced error message does not exist, reports in an unfriendly manner when `req.continue` is used incorrectly. No problem if no parameter is supplied, only if a parameter is provided *and* it isn't a function.

### Steps to test
1. Create a Cypress test with a `cy.intercept` that will trigger.
2. Add RouteHandler to intercept, use `req.continue({})` within the handler
```
cy.intercept('...', (req) => {
  req.continue({})
})
```
3. Run test, observe error referencing missing error message

### How has the user experience changed?
**Before**
![Screenshot 2023-02-20 at 11 36 30 AM](https://user-images.githubusercontent.com/11482842/220171703-6f3e8226-d878-4daa-84b3-59e37e47f11b.png)

**After**
![Screenshot 2023-02-20 at 11 43 23 AM](https://user-images.githubusercontent.com/11482842/220172125-de156c41-baeb-4b74-8a1c-168884ac52c6.png)

### PR Tasks
